src/EFCore.PG/Migrations/NpgsqlMigrationsSqlGenerator.cs

```diff
@@ -640,7 +640,8 @@ protected override void Generate(RenameTableOperation operation, IModel model, M
 
             builder.Append("INDEX ");
 
-            if (operation[NpgsqlAnnotationNames.CreatedConcurrently] is bool concurrently && concurrently)
+            var concurrently = (operation[NpgsqlAnnotationNames.CreatedConcurrently] as bool?).GetValueOrDefault();
```

Comment:
nit: Rewriting as shown below looks better, but generated worse IL.

```suggestion
            var concurrently = operation[NpgsqlAnnotationNames.CreatedConcurrently] as bool? == true;
```

Reply:
Definitely looks better, thanks! This code is completely perf-insensitive so we don't care about IL too much.